crates/bevy_sprite/src/dynamic_texture_atlas_builder.rs

```diff
@@ -43,14 +44,16 @@ impl DynamicTextureAtlasBuilder {
     pub fn add_texture(
         &mut self,
         atlas_layout: &mut TextureAtlasLayout,
+        textures: &mut Assets<Image>,
         texture: &Image,
-        atlas_texture: &mut Image,
+        atlas_texture_handle: &Handle<Image>,
```

Comment:
Is there some reason we need to undo #13717 here, or is this a merge artifact?

Reply:
Im assuming this is a merge artifact, this is a very old PR that has been dragged along

Reply:
and also I already merged a fix for this, getting confused by github...

Reply:
oh lol, to tired, this is what you mean by merge artifact?

Reply:
Sorry for the confusion. I added the review comment some time before sending the PR to fix it, which you merged.

By "merge artifact" I just mean any unintended change in the changeset that came about during the merge conflict resolution process.